### PR TITLE
data:Gautam-666

### PR DESCRIPTION
**Pull request by Gautam-666**
## Check List (Check all the applicable boxes) 

- [ ] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] All new and existing tests passed.
- [ ] This PR does not contain plagiarized content.
- [ ] The title of my pull request is a short description of the requested changes.

## Note to reviewers

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/5192"></a>

### CODE REVIEW

**Review by Pradumnasaraf — APPROVED** [review]
Looks good to merge. Welcome to LinkFree 🎉

File: data/Gautam-666.json
Comment (Pradumnasaraf) [inline data/Gautam-666.json:17]:
```suggestion
			"url": "https://www.instagram.com/gautam_rai06",
```

File: data/Gautam-666.json
Comment (Pradumnasaraf) [inline data/Gautam-666.json:27]:
```suggestion
			"url": "https://www.quora.com/profile/GAUTAM-RAI-163",
```

**Review by Pradumnasaraf — COMMENTED** [review]
Also, a small suggestion, you can remove the extra part in every URL. They are not required.

**Comment by SurajPratap10** [comment]
You put the same PR before also @Gautam-666  and then closed it after approval by members. Don't do this from now on as maintainers only can merge PR's , closing a PR by yourself won't merge it (just in case if you are new to OS and stuff I'm telling)..

**Comment by Gautam-666** [comment]
@SurajPratap10 ok i will keep this things in mind. Also since it was my first pull request, i will try to avoid mistakes which happened this time in upcoming pull req.

**Comment by Gautam-666** [comment]
@Pradumnasaraf I have applied your suggestions . Now what is the next step i should do . or is it done?

**Comment by SurajPratap10** [comment]
> @SurajPratap10 ok i will keep this things in mind. Also since it was my first pull request, i will try to avoid mistakes which happened this time in upcoming pull req.

No worries ;)

**Comment by SurajPratap10** [comment]
> @Pradumnasaraf I have applied your suggestions . Now what is the next step i should do . or is it done?

Now, maintainers will check in the end and merge, your job is done. Congrats on the 1st PR..

**Review by SurajPratap10 — APPROVED** [review]
LGTM

**Review by SaraJaoude — APPROVED** [review]
looks good

**Pull request merged**
